app/helpers/versions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module VersionsHelper
+  def version_date_tag(version)
+    data = {}
+    klass = ["gem__version__date"]
+    text = version_authored_date(version)
+    if version.rely_on_built_at?
+      klass << "tooltip__text"
```

Comment:
You should be add a modifier class to preserve the version number color. As per your screenshot, ones will tooltip are now red.

Reply:
Sounds reasonable! I wasn't sure if we want to preserve the color or highlight it to be aligned with other tooltips

I defined `.gem__version__date.tooltip__text` style explicitly to preserve the color from `gem__version__date`
Let me know if that is what you meant

<img width="472" alt="image" src="https://user-images.githubusercontent.com/5512772/180795857-81596523-35e7-42ee-be75-ddf1041285dd.png">

Reply:
This looks much better. Thank you.